### PR TITLE
Tutorials pablo

### PR DESCRIPTION
**Pull request by pgarciafluendo**

### CODE REVIEW

**Review by carlosfg-flu — CHANGES_REQUESTED** [review]
You should also squash both commits. All these changes should be in a single commit:

```bash
$ git rebase -i fluendo/
# edit the file
pick 7336baa8f3 prepare tutorials/ folder to be built with meson
fixup 99537b03f7 prepare tutorials/ to be built with meson
# save and close
$ git log -1 -p
# A single commit with all changes
```

File: subprojects/gst-docs/examples/tutorials/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Building Tutorials with Meson for VSCode
+
+Follow these steps to build the tutorials using Meson for VSCode:
```
Comment (carlosfg-flu) [inline subprojects/gst-docs/examples/tutorials/README.md:3]: These steps aren't really related to VSCode or any other IDE.

Comment (pgarciafluendo) [reply subprojects/gst-docs/examples/tutorials/README.md]: I thought I had also pushed the .vscode folder... but I now understand that this is not what should be done.

File: subprojects/gst-docs/examples/tutorials/meson.build [hunk 2]
```diff
@@ -0,0 +1,37 @@
+project('tutorials', 'c')
+
+dependencies = []
+dependencies += dependency('gstreamer-1.0')
+dependencies + = dependency('gtk+-3.0', version : '>=3.0')
+dependencies + = dependency('clutter-gst-3.0', version : '>=3.0')
+dependencies + = dependency('gstreamer-audio-1.0')
+dependencies + = dependency('gstreamer-pbutils-1.0')
```
Comment (carlosfg-flu) [inline subprojects/gst-docs/examples/tutorials/meson.build:8]: Meson fails to configure. There is an extra space between `+` and `=`.

```suggestion
dependencies += dependency('gstreamer-1.0')
dependencies += dependency('gtk+-3.0', version : '>=3.0')
dependencies += dependency('clutter-gst-3.0', version : '>=3.0')
dependencies += dependency('gstreamer-audio-1.0')
dependencies += dependency('gstreamer-pbutils-1.0')
```

**Review by carlosfg-flu — CHANGES_REQUESTED** [review]

File: subprojects/gst-docs/examples/tutorials/meson.build (hunk 2 above)
Comment (pgarciafluendo) [reply subprojects/gst-docs/examples/tutorials/meson.build]: Done

File: subprojects/gst-docs/examples/tutorials/.vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    "C_Cpp.default.compileCommands": "/home/pgarcia/fluendo/gstreamer/subprojects/gst-docs/examples/tutorials/builddir/compile_commands.json",
```
Comment (carlosfg-flu) [inline subprojects/gst-docs/examples/tutorials/.vscode/settings.json:2]: This is an absolute path from your personal computer. Would it be possible to use a variable or a relative path here?

Comment (pgarciafluendo) [reply subprojects/gst-docs/examples/tutorials/.vscode/settings.json:2]: Done

File: subprojects/gst-docs/examples/tutorials/.vscode/tasks.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+			"type": "meson",
+			"mode": "build",
+			"problemMatcher": [
+				"$meson-gcc"
+			],
+			"group": "build",
+			"label": "Meson: Build all targets"
+		}
+	]
+}
```
Comment (carlosfg-flu) [inline subprojects/gst-docs/examples/tutorials/.vscode/tasks.json:14]: It'd be good if you configure your editor to always put a newline at the end of files :slightly_smiling_face:

File: subprojects/gst-docs/examples/tutorials/.vscode/settings.json (hunk 3 above)
Comment (carlosfg-flu) [reply subprojects/gst-docs/examples/tutorials/.vscode/settings.json:2]: Missing `git push` maybe?

File: subprojects/gst-docs/examples/tutorials/.vscode/tasks.json (hunk 4 above)
Comment (carlosfg-flu) [reply subprojects/gst-docs/examples/tutorials/.vscode/tasks.json:14]: It also useful to highlight somehow the extra spaces at the end of line

Comment (pgarciafluendo) [reply subprojects/gst-docs/examples/tutorials/.vscode/tasks.json:14]: I though it was negative to leave an empty line at the end of files... at least that's what I understood for your "Extra empty lines at EOF" corrections a few commits ago

**Review by carlosfg-flu — APPROVED** [review]

**Comment by carlosfg-flu** [comment]
Take a look at how the community is formatting the commits related with this folder:

```
$ git log -10 --graph --oneline ./
* 8b3fbbc7ca add support for VSCode
* 03efe7c3b0 prepare tutorials/ folder to be built with meson
* 1db67b933a examples: Fix SSL not working in iOS tutorials
* edfd44d088 examples: iOS tutorial improvements
* ce81b81d3f examples: update ios deplyoment target to 12.0
* 9fdf3dbc7f docs: Port toolkit integration tutorial to gtksink
* 4aa3718c2d examples: iOS tutorials: remove kate plugin references here too
* 5889059cff examples/android: specify the exact NDK (r25c) version to use
* bbb0a30f32 docs: update old gstreamer-sdk media urls
* e0cf1ebff2 docs: iOS tutorials: update for pending ivorbisdec plugin removal
```

I don't know how strict they will be with this, but it'd be great to mimic their format.
```
$ git rebase -i HEAD~2
reword 03efe7c3b0 prepare tutorials/ folder to be built with meson
reword 7374166433 add support for VSCode
```

**Pull request merged**